### DISCUSSION TITLE
关于Select的Options异步加载的时候，默认值闪一下

### DISCUSSION THREAD

**Discussion by iHaroro** [ant-design/ant-design#36540] [Ideas]
最近用antd有个关于设计上问题

问题：Select的Options异步获取的情况下，然后defaultValues有value， 但是没有label，这时候Select的选项在Options异步未回来的时候时闪一下value，异步回来变成匹配到的label

想法：为什么不考虑在Options匹配时展示呢？未匹配则不展示value，不知道这个想法是不是有意义

**Comment by yoyo837** [comment]
会误以为defaultValues无效吧，用条件渲染就可以了，或者loading控制一下

**Reply by iHaroro** [reply]
可能这只是产品设计上的策略吧，毕竟这种东西也不算bug
